websites/R/ReaperX/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+    clientId: "751384892570468404"
+});
+  
+const browsingStamp = Math.floor(Date.now() / 1000);
+const path = document.location.pathname;

```

Comment:
combine consts

Reply:
sorry , What do you mean combine const am so confusing

Reply:
```
const var1 = val;
const var2 = val;
```

To:

```
const var1 = val,
          var2 = val;
```

Reply:
Is it correct that i upload it?